tests/README.md

```diff
@@ -14,7 +14,7 @@ SWSS, Redis, and all the other required components run inside a virtual switch D
     ```
     sudo modprobe team
     sudo apt install net-tools ethtool
-    sudo pip install docker pytest flaky redis
+    sudo pip install docker zipp==2.2.1 pytest==4.6.9 pytest flaky redis
```

Comment:
`pytest` is listed twice here.

Reply:
should be fixed now. Thanks for the report!